packages/devtools/src/common/bytes.ts

```diff
@@ -60,7 +60,7 @@ export const isZero = (value: PossiblyBytes | PossiblyBigInt | null | undefined)
  * @returns {string | undefined}
  */
 export const ignoreZero = <T extends PossiblyBytes | PossiblyBigInt>(value?: T | null | undefined): T | undefined =>
-    isZero(value) ? undefined : value ?? undefined
+    isZero(value) ? undefined : (value ?? undefined)
```

Comment:
Is there a reason this was added?

Reply:
looks like its autogenerated by `pnpm lint:fix`

Reply:
I'm not sure how, but your local version had discrepancies with main, I ran:

`git checkout main ./packages/devtools/src/common/bytes.ts`

and everything is good now.